packages/@internationalized/date/docs/ZonedDateTime.mdx

```diff
@@ -0,0 +1,486 @@
+
+This example creates a date in the Japanese calendar system, which is equivalent to April 4th, 2020 at 9:15 AM in the Gregorian calendar.
+
+```tsx
+import {JapaneseCalendar} from '@internationalized/date';
+
+let date = new ZonedDateTime(
+  new JapaneseCalendar(), 'heisei', 31, 4, 30,
+  'America/Los_Angeles', -28800000,
+  9, 15
+);
+```
+
+A list of valid era identifiers can be retried using the `getEras` method of a `Calendar` instance. If an era is not provided, the date is assumed to be in the current era.
+
+### Converting between calendars
+
+The <TypeLink links={docs.links} type={docs.exports.toCalendar} /> function can be used to convert a date from one calendar system to another.
+
+This example converts a Gregorian date to a Hebrew one.
+
+```tsx
+import {toCalendar, HebrewCalendar} from '@internationalized/date';
+
+let gregorianDate = new ZonedDateTime(
+  2020, 9, 19,
+  'America/Los_Angeles', -28800000,
+  10, 30
+);
+
+let hebrewDate = toCalendar(gregorianDate, new HebrewCalendar());
+```
+
+## Manipulating dates
+
+### Adding and subtracting durations
+
+A <TypeLink links={docs.links} type={docs.exports.DateTimeDuration} /> is an object that represents an amount of time, with fields such as `years`, `months`, `hours`, and `minutes`. The `add` and `subtract` methods of `CalendarDateTime` objects can be used to adjust the date by the given duration. These methods return a new date, and do not mutate the original.
+
+```tsx
+let date = parseZonedDateTime('2022-02-03T09:45[America/Los_Angeles]');
+
+date.add({weeks: 1}); // 2022-02-09T09:45[America/Los_Angeles]
+date.add({months: 1}); // 2022-03-03T09:45[America/Los_Angeles]
+date.add({years: 1, months: 1, days: 1}); // 2023-03-04T09:45[America/Los_Angeles]
+date.add({hours: 1}); // 2022-02-03T10:45[America/Los_Angeles]
+date.add({minutes: 30}); // 2022-02-03T10:15[America/Los_Angeles]
+
+date.subtract({weeks: 1}); // 2022-01-27T09:45[America/Los_Angeles]
+date.subtract({months: 1}); // 2022-01-03T09:45[America/Los_Angeles]
+date.subtract({years: 1, months: 1, days: 1}); // 2021-01-02T09:45[America/Los_Angeles]
+date.subtract({hours: 1}); // 2022-02-03T8:45[America/Los_Angeles]
+date.subtract({minutes: 30}); // 2022-02-03T09:15[America/Los_Angeles]
+```
+
+Adding or subtracting a duration that goes beyond the limits of a particular field will cause the date to be _balanced_. For example, adding one day to August 31st results in September 1st. In addition, if adding or subtracting one field causes another to be invalid, the date will be _constrained_. For example, adding one month to August 31st results in September 30th because September 31st does not exist.
```

Comment:
I'm looking at the next paragraph and wondering if we should use quotes instead of italics. I'm concerned about another formatting style to "confuse" readers and another rule for us to follow.